app/models/facility.rb

```diff
@@ -17,12 +17,10 @@ class Facility < ApplicationRecord
   has_many :encounters
   has_many :blood_pressures, through: :encounters, source: :blood_pressures
   has_many :blood_sugars, through: :encounters, source: :blood_sugars
-  has_many :patients, -> { distinct }, through: :encounters
+  has_many :patients, -> { distinct }, through: :blood_pressures
```

Comment:
This is the first change made to this file.

Reply:
Why are we modifying this relationship? Since we now deal with BPs and BSs, should we instead add a second association like `facility.blood_pressure_patients`?

Reply:
This is a good point.

Added a new association for [this](https://github.com/simpledotorg/simple-server/pull/888/commits/14fb637fd5b072dcce41e97afc5184c20ca63d8d).

The reason why I originally did it because I thought we'd naturally want to assume hypertensive as the default case. But it makes sense to be explicit instead.

There's a `diabetes_patients` scope in the upcoming [DM/HTN split in progress tab](https://github.com/simpledotorg/simple-server/pull/897).